orderer/common/multichannel/registrar.go

```diff
@@ -155,8 +160,57 @@ func NewRegistrar(
 func (r *Registrar) Initialize(consenters map[string]consensus.Consenter) {
 	r.consenters = consenters
 	existingChannels := r.ledgerFactory.ChannelIDs()
+	existingChannelsMap := map[string]struct{}{}
+	for _, channel := range existingChannels {
+		existingChannelsMap[channel] = struct{}{}
+	}
```

Comment:
Following the code review we had, the registrar struct and registrar.Initialize are going to change a lot even before this PR,  so I would advise waiting until the dust settles. 

Among other things, we're gonna have a separate new map for followers and are going to need to discover if the system channel exists first.

Reply:
This was handled via @tock-ibm 's refactor of the registrar initialization.